mobile/lib/routing/auth_guard.dart

```diff
@@ -16,28 +17,31 @@ class AuthGuard extends AutoRouteGuard {
     resolver.next(true);
```

Comment:
This is the only section I'm not terribly sure about. I think we should block the resolver if `Store.get(StoreKey.accessToken)` excepts with a `StoreKeyNotFoundException` because we do that synchronously. But I didn't want to change the logic here, so I left this intact.